package.json

```diff
@@ -29,6 +29,8 @@
   ],
   "author": "Matt Roberts",
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
The dependencies should be in the corresponding package (I believe in `./packages/concerto-ui-react/package.json` ?) rather than at the top-level.

Reply:
It should also use the same version of `moment` as the one used in Concerto:
https://github.com/accordproject/concerto/blob/c7ee9cbe6487ae8e590f608993053dee5abf6694/packages/concerto-core/package.json#L70